docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,293 @@
+
+============================
+Getting started with Flocker
+============================
+
+.. logo::
+
+.. intro-text::
+
+   Flocker lets you run microservices apps with database containers and move them around between servers. It comes in two pieces and you’ll need both.
+
+.. contents::
+   :local:
+
+.. tutorial-step::
+
+   Step 1: Installing Flocker CLI & Node
+   =====================================
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+      .. image:: images/macbook.png
+         :class: center-block img-responsive
+         :alt: Flocker CLI diagram
+
+      Flocker CLI
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Local
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            you@laptop:~$ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Ubuntu / Debian
+         ^^^^^^^^^^^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \         
+              virtualenv flocker-tutorial && \
```

Comment:
Please file a follow-up issue for moving the `activate ...` to immediately after this line and changing the `pip` invocations on the subsequent lines to rely on `${PATH}` instead of specifying a path.